From b0602462ff8196422908c90cfeac33a0b128c97c Mon Sep 17 00:00:00 2001
From: Ezra Buehler <ezra.buehler@husqvarnagroup.com>
Date: Thu, 11 Jul 2024 17:49:36 +0200
Subject: [PATCH] bootcount: Allow boot counter to be enabled for AT91SAM9X5

The driver is compatible with similar SoCs like the AT91SAM9X5 family.
This has been tested on the AT91SAM9G25-based GARDENA smart Gateway.

Signed-off-by: Ezra Buehler <ezra.buehler@husqvarnagroup.com>
Upstream-Status: Pending
---
 drivers/bootcount/Kconfig | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/bootcount/Kconfig b/drivers/bootcount/Kconfig
index b5ccea0d9c..828ac90ffe 100644
--- a/drivers/bootcount/Kconfig
+++ b/drivers/bootcount/Kconfig
@@ -13,7 +13,7 @@ if BOOTCOUNT_LIMIT
 choice
 	prompt "Boot count device"
 	default BOOTCOUNT_AM33XX if AM33XX || SOC_DA8XX
-	default BOOTCOUNT_AT91 if AT91SAM9XE
+	default BOOTCOUNT_AT91 if AT91SAM9XE || AT91SAM9X5
 	default BOOTCOUNT_GENERIC
 
 config BOOTCOUNT_GENERIC
@@ -69,8 +69,8 @@ config BOOTCOUNT_I2C
 	                              the bootcounter.
 
 config BOOTCOUNT_AT91
-	bool "Boot counter for Atmel AT91SAM9XE"
-	depends on AT91SAM9XE
+	bool "Boot counter for Atmel AT91SAM9XX"
+	depends on AT91SAM9XE || AT91SAM9X5
 
 config DM_BOOTCOUNT
         bool "Boot counter in a device-model device"
-- 
2.43.0

